### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by hoangphan84**
Add footer with build version and style following SSW home page

### CODE REVIEW

File: src/components/footer/footer.js [hunk 1]
```diff
@@ -1,17 +1,83 @@
...
+				<div className="w-1/6" />
+				<div className="w-1/6">
+					Copyright © SSW 1990 - {new Date().getFullYear()}. All Rights
+					Reserved.
+				</div>
+				<div className="w-3/6 text-right">
+					<a className="text-gray-800" href="mailto:info@ssw.com.au">
+						FEEDBACK TO SSW
+					</a>
+					&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;
```
Comment (JeanThirion) [inline src/components/footer/footer.js]: Could you please use a spacer class with paddings instead ?
Make sure you use "em" sizes so it scales with zooms as well

File: src/style.css [hunk 2]
```diff
@@ -61,15 +61,20 @@ a:link:not(.btn):not(.unstyled):hover {
...
 }
 
-footer > a:not(.unstyled) {
-  @apply text-gray-300 !important;
+footer > div > a:not(.unstyled), footer > div > div > a:not(.unstyled)  {
+  @apply text-gray-300 !important
 }
 
-footer > a:not(.unstyled):hover {
+footer > div > a:not(.unstyled):hover, footer > div > div > a:not(.unstyled):hover {
```
Comment (JeanThirion) [inline src/style.css]: it looks like a pretty complicated selector, any chance we could do better ?

File: src/components/footer/footer.js (hunk 1 above)
Comment (hoangphan84) [reply src/components/footer/footer.js]: Done

File: src/style.css (hunk 2 above)
Comment (hoangphan84) [reply src/style.css]: Done

**Pull request merged**